### PR TITLE
todo list edit update unit testing part 2

### PR DESCRIPTION
**Pull request by codecaiine**
**In this pull request these are the following requirements:**

 -   a function for editing the task description.
 -   a function for updating an item's 'completed' status.
 -   a function for updating an item's index value upon drag/drop actions.
 -   the "Clear all completed" function.

### CODE REVIEW

File: tests/edit-update.test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    check(checkbox, tasks[0]);
+    expect(tasks[0].completed).toBeFalsy();
+  });
+  test("test the last task if it's checked ", () => {
+    addTask(inputList);
+    const tasks = getStorage();
+    checkbox.checked = true;
+    check(checkbox, tasks[tasks.length - 1]);
+    expect(tasks[tasks.length - 1].completed).toBeTruthy();
+  });
```
Comment (george-shammar) [inline tests/edit-update.test.js:67]: - [ ] Please, let's test the "Clear all completed" function according to the project's requirement.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @codecaiine and @Kossi-stack , 

Good job so far!

:heavy_check_mark: You have created a professional repository. YOur commit messages, PR and linters are well done. :clap: 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Not Applicable

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: tests/edit-update.test.js (hunk 1 above)
Comment (codecaiine) [reply tests/edit-update.test.js]: Hi,
Thank you for reviewing our work. We made some updates. As you can see we test the "Clear all completed" function here.

demoooo111

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Congratulations🎉 🎉 

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: tests/edit-update.test.js (hunk 1 above)
Comment (codecaiine) [reply tests/edit-update.test.js]: Also  for "Clear all completed" function. I just wanna mention that this function is named removeCompleted .

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

### Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DeliceLydia** [comment]
# Hi Team

your project is approved I am the one who made a mistake on the first review
by asking to make a change but I approved it!

Sorry for the inconvenience!

Lydia

**Pull request merged**